package.json

```diff
@@ -15,7 +15,8 @@
     "@mui/lab": "^5.0.0-alpha.92",
     "@mui/material": "^5.9.1",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-json-tree": "^0.17.0"
```

Comment:
normally you should put this dep into devDep not in peer.

only double instance dep are peerDep (mui with emotion and react)

other third party lib should remain wrapped inside the buid from devDep